__tests__/app.test.js

```diff
@@ -26,3 +26,39 @@ describe("GET /api/topics", () => {
       });
   });
 });
+describe("GET /api/articles/:article_id", () => {
+  test("200: responds with the article matching the input article id passed as a parameter", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toEqual(
+          expect.objectContaining({
+            article_id: expect.any(Number),
```

Comment:
At the moment, this test would pass even if the wrong object came back. 
You'd need to check the article_id, is the same ID you're checking for, not just any number.